travis_test_migrations.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+git fetch origin master
```

Comment:
Is `git fetch` necessary? Or could we just `git checkout master` since Travis does a `git clone --depth 50`?

I saw that you `exit 0` only if the migration files match but... should this script have `set -e` in case any step fails (or some other "graceful failure")? Also, I seem to recall Dennis frowning on the use of `set -e`.